src/OrchardCore/OrchardCore.Autoroute.Core/AutorouteConstants.cs

```diff
@@ -0,0 +1,6 @@
+namespace OrchardCore.Autoroute;
+
+public class AutorouteConstants
+{
+    public const string HandlePrefix = "slug:";
```

Comment:
Shouldn't this be called SlugPrefix instead?

Reply:
Could be